src/index.ts

```diff
@@ -1627,61 +1648,67 @@ const configShardusEndpoints = (): void => {
     }
 
     const address = req.params['address']
+    try {
+      const accountData = await getAccountData(address, req)
+      res.json(accountData)
+    } catch (error) {
+      res.json({ error: error.message || 'An error occurred' })
+    }
+  })
+
+  async function getAccountData(address, req) {
```

Comment:
can this function be moved to a util folder ? it also has some check failures 
<img width="387" alt="image" src="https://github.com/user-attachments/assets/559a6d99-6ac0-4c47-afc9-a3a054714157">

Reply:
moved to utils and added typing